core/src/main/java/org/hisp/dhis/android/network/bing/BingService.kt

```diff
@@ -26,17 +26,17 @@
  *  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package org.hisp.dhis.android.core.map.layer.internal.bing
+package org.hisp.dhis.android.network.bing
 
-import org.hisp.dhis.android.core.arch.api.HttpServiceClient
+import org.hisp.dhis.android.network.common.HttpServiceClientKotlinx
 import org.koin.core.annotation.Singleton
 
 @Singleton
```

Comment:
We can do a general review of the Services to check if they have the Singletion annotation, no need to remove it now